_security/configuration/opensearch-keystore.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: OpenSearch keystore
```

Comment:
Global: OpenSearch keystore or OpenSearch Keystore? I'll use the first for now but let me know.

Reply:
Switched this to capitalize Keystore for consistency.

Reply:
It should only be capitalized as a proper noun, in other words, if this is a formally named feature that is specific to OpenSearch.

Reply:
@natebower wdyt in this case? This is one of the scripts that is shipped with [opensearch](https://github.com/opensearch-project/OpenSearch/tree/main/distribution/src/bin). Here's another [page](https://opensearch.org/docs/latest/install-and-configure/plugins/) on the documentation-website that details the `opensearch-plugin` script.

Reply:
Based on the content at the links, I would probably use "OpenSearch keystore" as the title and heading and when referencing _the_ OpenSearch keystore and `opensearch-keystore` (as you've done above) when referencing the name of the script.

Reply:
👍 I updated all instances from OpenSearch Keystore to OpenSearch keystore in this doc.